src/classes/RecipeRepository.js

```diff
@@ -1,6 +1,18 @@
+import Recipe from './Recipe';
+
 class RecipeRepository {
-  constructor(recipes = []) {
-    this.recipes = recipes;
+  constructor(recipes) {
```

Comment:
Nicely done merging all the data into the class files. This cleans up our scripts file and keeps all the data in the RecipeRepository class.